pkg/controller/labelidentity/label_group_index_test.go

```diff
@@ -42,6 +42,7 @@ var (
 	selectorC     = types.NewGroupSelector("", pSelDB, nil, nil, nil)
 	selectorD     = types.NewGroupSelector("testing", pSelDB, nil, nil, nil)
 	selectorE     = types.NewGroupSelector("random", pSelDB, nil, nil, nil)
+	selectorF     = types.NewGroupSelector("", nil, nsSelTest, nil, nil)
```

Comment:
May not related to this issue. But I think it would be better to also have a selector only have a namespace to cover more cases.
```
selectorG = types.NewGroupSelector("testing", nil, nil, nil, nil)
```